dpgen/data/gen.py

```diff
@@ -627,8 +646,10 @@ def run_vasp_md(jdata, mdata):
     run_tasks = [ii.replace(work_dir+"/", "") for ii in md_run_tasks]
     #dlog.info("md_work_dir", work_dir)
     #dlog.info("run_tasks",run_tasks)
-    dispatcher = make_dispatcher(mdata['fp_machine'], mdata['fp_resources'], work_dir, run_tasks, fp_group_size)
-    dispatcher.run_jobs(fp_resources,
+    api_version = mdata.get('api_version', '0.9')
+    if LooseVersion(api_version) < '1.0':
```

Comment:
Isn't it the following?
```suggestion
    if LooseVersion(api_version) < LooseVersion('1.0'):
```